langchain/examples/job_scraper/README.md

```diff
@@ -0,0 +1,39 @@
+# Job Scraper
+
+This agent extracts all of the job postings and their information from https://www.ycombinator.com/jobs and writes it in a new json file `job_postings.json`.
+
+## Run the script
+
+- Install LangChain and LangGraph to build the agent:
+
+```
+pip install langchain langgraph
+```
+
+- Install OpenAI as the language model used for this script:
+
+```
+pip install -qU "langchain[openai]"
+```
+
+- Install AgentQL's tools:
+
+```
+pip install -U langchain-agentql
```

Comment:
You can gather all pip installs in one command for quicker setup. Imagine yourself being a user and needing to jump back and forth 3 times to copy-paste instead of 1

Reply:
That's a good point